.github/workflows/impress-frontend.yml

```diff
@@ -130,7 +131,6 @@ jobs:
           DOCKER_BUILDKIT: 1
           COMPOSE_DOCKER_CLI_BUILD: 1
         run: |
-          docker-compose build --pull --build-arg BUILDKIT_INLINE_CACHE=1
```

Comment:
can you explain why we must remove this?

Reply:
Yes, not related, I will remove the change.